src/js/components/invoice/AddItemsPage.jsx

```diff
@@ -340,7 +341,7 @@ class AddItemsPage extends Component {
   saveAndExit(formValues) {
     this.saveInvoiceItems(formValues)
       .then(() => {
-        window.location = stringUrlInterceptor(`/invoice/show/${formValues.id}`);
+        window.location = stringUrlInterceptor(INVOICE_URL.show(formValues.id));
```

Comment:
You said in the description that we have to use `stringUrlInterceptor` for redirecting to list pages. Maybe in the redirect URLs, we can include the host name in the URL in the same way as we are passing `CONTEXT_PATH`?

Or maybe use the interceptor in the URL generator instead of using it on each page?

Reply:
I think I like the second suggestion best.